### PR TITLE
Competition indexing

### LINKED ISSUE #470: subgraph should supply `CompetitionProposal.admin`
It appears that the subgraph is not supplying `CompetitionProposal.admin` as it should be.  Lacking this value prevents Alchemy from correctly disabling the New Submission button.

This won't compile:

```
{
  proposals {
    id, competition {
      admin
    }
  }
}
```

here:  https://thegraph.com/explorer/subgraph/daostack/v37_3_rinkeby?version=current, which is what Alchemy rinkeby is currently pointed at.

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #470 #476

### CODE REVIEW

File: src/mappings/Competition/mapping.ts [hunk 1]
```diff
@@ -178,6 +180,7 @@ export function handleNewVote(event: NewVote): void {
     }
     competitionProposal.winningSuggestions = winningSuggestions;
     competitionProposal.totalVotes = competitionProposal.totalVotes.plus(BigInt.fromI32(1));
+    competitionProposal.numWinningSubmissions = BigInt.fromI32(winningSuggestions.length);
     competitionProposal.save();
```
Comment (dkent600) [inline src/mappings/Competition/mapping.ts]: we should be consistent here with naming submissions and suggestions

Comment (ben-kaufman) [reply src/mappings/Competition/mapping.ts]: But this is the name you asked it to have. So should I change it to `numWinningSuggestions`?

Comment (ben-kaufman) [reply src/mappings/Competition/mapping.ts]: I changed it for consistency

File: src/mappings/Competition/mapping.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export function handleNewCompetitionProposal(event: NewCompetitionProposal): voi
   competitionProposal.winningSuggestions = [];
   competitionProposal.totalSubmissions = BigInt.fromI32(0);
```
Comment (dkent600) [inline src/mappings/Competition/mapping.ts]: should be totalSuggestions

File: src/mappings/Competition/mapping.ts (hunk 1 above)
Comment (dkent600) [reply src/mappings/Competition/mapping.ts]: We are constantly getting confused about "suggestions" and "submissions", hence my error n the request.  Sorry about that.  But subgraph, I think, should itself be consistent.

File: src/mappings/Competition/schema.graphql [hunk 3]
```diff
@@ -16,8 +16,10 @@ type CompetitionProposal @entity {
     winningSuggestions: [CompetitionSuggestion!]
     votes: [CompetitionVote!] @derivedFrom(field: "proposal")
     createdAt: BigInt!
-    totalSubmissions: BigInt!
+    totalSuggestions: BigInt!
     totalVotes: BigInt!
+    numWinningSuggestions: BigInt!
```
Comment (dkent600) [inline src/mappings/Competition/schema.graphql]: Perhaps to be consistent with other props here, should be `numberOfWinningSuggestions`

Comment (ben-kaufman) [reply src/mappings/Competition/schema.graphql]: ok...

**Pull request merged**